bindings/python/src/tokenizer.rs

```diff
@@ -1165,15 +1171,15 @@ mod test {
     use super::*;
     use crate::models::PyModel;
     use crate::normalizers::{PyNormalizer, PyNormalizerTypeWrapper};
-    use std::sync::Arc;
+    use std::sync::{Arc, RwLock};
     use tempfile::NamedTempFile;
     use tk::normalizers::{Lowercase, NFKC};
 
     #[test]
     fn serialize() {
-        let mut tokenizer = Tokenizer::new(PyModel::new(Arc::new(
+        let mut tokenizer = Tokenizer::new(PyModel::new(Arc::new(RwLock::new(
```

Comment:
Could you explain why all those RwLock are need ? I'm not sure why we would need them.

Reply:
Sure, we need them because anything in an `Arc` is immutable. These `RwLock` provides us with a way to actually mutate the `Model` here.